operations/admin_from_mdb.go

```diff
@@ -0,0 +1,282 @@
+package operations
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/evergreen-ci/birch"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
wrong bson library, though in this context it's probably safe to use birch exclusively?

Reply:
Switched to go driver.